v4/integrations/nrawssdk-v1/nrawssdk_test.go

```diff
@@ -98,7 +98,7 @@ var (
 		},
 	}
 	externalSpanNoRequestID = internal.WantSpan{
-		Name:     "http POST unknown",
+		Name:     "http POST lambda.us-west-2.amazonaws.com",
```

Comment:
#### Bug 1

This bug was uncovered by the `External` metrics. In those we were specifically looking for this host. That made me realize that we _should_ be collecting the host, but were not. This was because the host was set in the `Request.URL` which we were not checking as part of creating the span name.